src/cli/backup/helpers_test.go

```diff
@@ -132,6 +133,7 @@ type dependencies struct {
 func prepM365Test(
 	t *testing.T,
 	ctx context.Context, //revive:disable-line:context-as-argument
+	pst path.ServiceType,
```

Comment:
nit: can we name this `service` please? The current variable name is still referring to the type, not the actual usage of the thing

Reply:
"service" is also a reference to the type and not its usage, is it not?  Just a more human way of writing it.

Reply:
service also describes what it is. `pst` alone gives no context as to what it's referring to. For all I know it's talking about files that you hook up to Exchange to read emails, contacts, and calendar events 😛